subiquity/models/filesystem.py

```diff
@@ -1115,6 +1115,8 @@ class ZPool:
     # default dataset options for the zfses in the pool
     fs_properties: Optional[dict] = None
 
+    default_features: Optional[bool] = True
```

Comment:
Er this doesn't seem to match the change to made to curtin in the end. Did you forget to push an update to this branch?

Reply:
No, I think this is correct, just confusing (and this is the less confusing version!)
See also the matching curtin side, https://git.launchpad.net/curtin/commit/?id=4195867e55953afd8c2481f813db3cda17791ead

in the simplest `zpool create $poolname $device` case, no `-d` arg is present and we include the default features.
If we want maximum control, we can pass `-d`, default_features=False, and then supply feature flags that we want.

Reply:
oh ok